example/todomvc-flow/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "todomvc-flow",
+  "version": "1.0.0",
+  "description": "TODO app example",
+  "private": true,
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "clean": "rimraf public/build && mkdirp public/build",
+    "start": "webpack-dev-server --hot --content-base public/ --open",
+    "build": "NODE_ENV=production webpack -p",
+    "watch": "NODE_ENV=development webpack -d -w",
+    "prepublish": "npm run --if-present build",
+    "test": "mocha"
```

Comment:
Can you add flow check script?
It means that CI always test type definition.

```
"scripts": {
  "test": "mocha && npm run flow_check"
  "flow_check": "flow check"
}
// via https://flowtype.org/blog/2016/08/01/Windows-Support.html
```

[DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped "DefinitelyTyped") of TypeScript has a set of d.ts and test code for d.ts.
I think that example project and flow check is instead of this.